v2/azure-arm.yaml

```diff
@@ -371,6 +371,18 @@ typeTransformers:
     remove: true
     because: it is supposed to be marked ReadOnly in Swagger
 
+  - group: machinelearningservices
+    name: SystemCreatedStorageAccount
+    property: ArmResourceId
+    remove: true
+    because: it is supposed to be marked ReadOnly in Swagger
```

Comment:
Would we want to publish these as configmaps for people to consume?

If we do, maybe we need a new type-transformation to mark as read-only, so it ends up only on the `status`.

Reply:
I think this transformation actually does that, because by the time typeTransformers runs I'm pretty sure types are forked into STATUS and non-status. Removing it here on the non-status only removes it from the spec.

See for example:
```
  - group: machinelearningservices
    name: EncryptionProperty
    property: Status
    remove: true
    because: it is supposed to be marked ReadOnly in Swagger
```

but that field does exist on the status type still:

```
type EncryptionProperty_STATUS struct {
	// Identity: The identity that will be used to access the key vault for encryption at rest.
	Identity *IdentityForCmk_STATUS `json:"identity,omitempty"`

	// KeyVaultProperties: Customer Key vault properties.
	KeyVaultProperties *KeyVaultProperties_STATUS `json:"keyVaultProperties,omitempty"`

	// Status: Indicates whether or not the encryption is enabled for the workspace.
	Status *EncryptionProperty_Status_STATUS `json:"status,omitempty"`
}
```